content/en/docs/tasks/debug-application-cluster/applications/_index.html

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
this should be _index.md ?

Reply:
move this whole section one level up, and name it as "debug-applications"?

Reply:
If we move the application / cluster sections up a level, what about the `troubleshooting.md` doc? It's sort of a general info doc, so probably doesn't need its own high-level section, so maybe it makes sense to organize like this:

```
tasks/
   |--- troubleshooting.md
   |--- debug-application/
      |--- debug-pods.md
      |--- (other application docs...)
   |--- debug-cluster/
      |--- debug-cluster.md
      |--- (all the docs currently under monitoring/...)
```

Open to other suggestions as well, that's just what initially came to mind...

Reply:
We may need to investigate the contents in `troubleshooting.md` and split them accordingly. Troubleshooting a cluster is completely a different topic than troubleshooting an application or a Pod.

Reply:
Checked the contents in the `troubleshooting.md` file. I agree with you that it is a generic page. So your proposal makes good sense.